packages/app/background-jobs-common/src/periodic-jobs/AbstractPeriodicJob.ts

```diff
@@ -0,0 +1,301 @@
+import { randomUUID } from 'node:crypto'
+
+import {
+  type CommonLogger,
+  type ErrorReporter,
+  type TransactionObservabilityManager,
+  isError,
+} from '@lokalise/node-core'
+import { resolveGlobalErrorLogObject } from '@lokalise/node-core'
+import type Redis from 'ioredis'
+import { stdSerializers } from 'pino'
+import type { LockOptions } from 'redis-semaphore'
+import { Mutex } from 'redis-semaphore'
+import type { LockLostCallback } from 'redis-semaphore'
+import type { ToadScheduler } from 'toad-scheduler'
+import { AsyncTask, SimpleIntervalJob } from 'toad-scheduler'
+
+const DEFAULT_JOB_INTERVAL = 60000
```

Comment:
Does it mean that by default job will be executed once per minute? could we put a comment to clarify it 🙏

Reply:
after thinking about it some more, I've removed the default, as every job is going to be different, having arbitrary default is going to be more harmful than useful

Reply:
I agree with that! thanks Igor 🙏